### PR TITLE
Split live testing YAML to allow separate live testing of services

### PR DESCRIPTION
**Pull request by danieljurek**
- Updated `client.yml` to run only non-live tests
- Created `live.yml` to include only live tests and removed jobs not relevant to live testing

### CODE REVIEW

File: .azure-pipelines/live.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          arguments: $(OSName)
+
+      - task: DotNetCoreInstaller@0
+        displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
+        inputs:
+          version: '$(DotNetCoreVersion)'
+
+      - task: DotNetCoreCLI@2
+        displayName: 'Build & Test (live tests)'
+        condition: ne(variables['System.TeamProject'], 'public')
```
Comment (weshaggard) [inline .azure-pipelines/live.yml]: While it does not hurt anything this shouldn't be needed.

File: .azure-pipelines/live.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        env:
+          DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+          DOTNET_CLI_TELEMETRY_OPTOUT: 1
+          DOTNET_MULTILEVEL_LOOKUP: 0
+          APP_CONFIG_CONNECTION: $(AppConfigConnectionString)
+          AZ_SERVICE_BUS_CONNECTION: $(ServiceBusConnectionString)
+        inputs:
+          command: test
+          projects: '$(ProjectFile)'
+          arguments: --filter TestCategory=Live --logger trx
```
Comment (weshaggard) [inline .azure-pipelines/live.yml]: Remove category and run everything.

File: .azure-pipelines/live.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# External variables:
```
Comment (weshaggard) [inline .azure-pipelines/live.yml]: lets name this tests.yml since it is running all tests.

**Comment by weshaggard** [comment]
cc @Azure/azure-sdk-eng 

FYI @AlexGhiondea

File: .azure-pipelines/tests.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
+ernal variables:
```
Comment (weshaggard) [inline .azure-pipelines/tests.yml]: Looks like you accidentally deleted the beginning of this comment.

Comment (danieljurek) [reply .azure-pipelines/tests.yml]: 🤦‍♂️

**Review by weshaggard — APPROVED** [review]
LGTM - Please squash and merge. Please be ready to hook up the internal pipelines soon after merging.

**Comment by weshaggard** [comment]
@dsgouda the travis job has been failing today with `The command "sudo apt-get update" failed and exited with 100 during .` is that a known issue? This PR shouldn't impact the travis job so we are going to ignore it here.

**Comment by dsgouda** [comment]
> 
> 
> @dsgouda the travis job has been failing today with `The command "sudo apt-get update" failed and exited with 100 during .` is that a known issue? This PR shouldn't impact the travis job so we are going to ignore it here.

@weshaggard Feels like an intermittent issue, I have restarted the build to be sure

**Comment by dsgouda** [comment]
Still investigating the issue, could be a travis error. Confirming this change won't affect our builds. Feel free to approve and merge

**Pull request merged**